cart/views.py

```diff
@@ -0,0 +1,53 @@
+from django.http import JsonResponse
+from django.urls import reverse_lazy
+from django.views import View
+from django.views.generic import DetailView, ListView, RedirectView, UpdateView
+
+from cart.models import Cart
+from product.models import Product
+
+
+class ViewCart(ListView):
+    model = Cart
+    template_name = 'cart/cart.html'
+    context_object_name = 'cart'
+
+    def get_queryset(self):
+        return Cart.objects.get(user__id=self.request.user.id)
+
+
+class AddProduct(View):
```

Comment:
GET not for saving!!!!!! https://www.w3schools.com/tags/ref_httpmethods.asp start with this article